### PR TITLE
Reviewed Docs for UI errors and section mistakes

### PR DESCRIPTION
**Pull request by dino-saurabh**
1. fixed response error
2. made base URLs uniform wherever possible
3. fixed some console language mismatches
4. checked all internal links and fixed broken links

### CODE REVIEW

**Comment by dino-saurabh** [comment]
@Akayeshmantha the reviewed section looks good now :) please review !!

**Comment by dino-saurabh** [comment]
fixing this 2 heading problem
 this was one of the errors that crept in. there were 2 headings then 2 body sections on console

File: source/includes/Person/person_attribute_type.md [hunk 1]
```diff
@@ -57,38 +90,6 @@ GET /personattributetype?q=race&v=default
...
-  headers: myHeaders,
-  redirect: 'follow'
-};
-
-fetch("/openmrs/ws/rest/v1/personattributetype?q=race&v=default", requestOptions)
-  .then(response => response.text())
-  .then(result => console.log(result))
-  .catch(error => console.log('error', error));
-
-```
```
Comment (dino-saurabh) [inline source/includes/Person/person_attribute_type.md:93]: this was the Issue the response has to be the last segment in the console and hence should come after all the laguage tab sections

**Review by Akayeshmantha — APPROVED** [review]
Thanks LGTM

**Pull request merged**